artifacts/camara-cloudevents/event-subscription-template.feature

```diff
@@ -0,0 +1,356 @@
+    And the response header "Content-Type" is "application/json"
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_32_get_unknown_<xxx>_subscription_for_a_device
+  Scenario:  Get method for <xxx> subscription with subscription-id unknown to the system  
+    Given the path parameter "subscriptionId" is set to the value unknown to system
+    When the request "retrieve<xxx>Subscription" is sent
+    Then the response  code is 404
+    And the response property "$.status" is 404
+    And the response property "$.code" is "NOT_FOUND"
+    And the response property "$.message" contains a user friendly text
+
+########################### Subscription deletion scenarios #####################################
+
+  @<xxx>_subscriptions_33_no_authorization_header_for_delete_subscription
+  Scenario: No Authorization header for delete subscription
+    Given header "Authorization" is set without a token
+    When the request "delete<xxx>Subscription" is sent 
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_34_expired_access_token_for_delete_subscription
+  Scenario: Expired access token for delete subscription
+    Given header "Authorization" is set with an expired token
+    When the request "delete<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_35_invalid_access_token_for_delete_subscription
+  Scenario: Invalid access token for delete subscription
+    Given header "Authorization" set to an invalid access token
+    When the request "delete<xxx>Subscription" is sent
+    Then the response status code is 401
+    And the response header "Content-Type" is "application/json"
+    And the response property "$.status" is 401
+    And the response property "$.code" is "UNAUTHENTICATED"
+    And the response property "$.message" contains a user friendly text
+
+  @<xxx>_subscriptions_36_delete_invalid_<xxx>_subscription
+  Scenario:  Delete <xxx> subscription with subscription-id unknown to the system
+    Given the path parameter "subscriptionId" is set to the value unknown to system
+    When the request "delete<xxx>Subscription" is sent
+    Then the response code is 404
+    And the response property "$.status" is 404
+    And the response property "$.code" is "NOT_FOUND"
+    And the response property "$.message" contains a user friendly text
+
+########Specific Subscription error scenarios when Device object is  used ################

```

Comment:
As we have these scenarios included in dedicated artifacts we may just reference the other artifacts to not maintain equivalent scenarios in several places

Reply:
Yes agree with your proposal @jlurien  to remove them to avoid redundancy.
Before to remove them looking for OK from other "usual suspects": @PedroDiez @rartych @patrice-conil  OK for you?

Reply:
Yes, I have updated #479, so now artifact is compatible with APIs having device or phoneNumber in paths other than $.xxx